js/main.js

```diff
@@ -0,0 +1,109 @@
+(function() {
+    function loadPics() {
+        var api_key, method, user_id, format, url, xmlhttp;
+
+        api_key = '74e47a159e15cbcb6139ba9c9df64c13';
+        method = "flickr.photos.search";
+        user_id = "138698049@N03";
+        format = '&format=json';
+        url = "https://api.flickr.com/services/rest"+"?method="+method+"&api_key="+api_key+"&user_id="+user_id+format+'&nojsoncallback=1';
+        xmlhttp = new XMLHttpRequest();
+
+        xmlhttp.onreadystatechange = function () {
+            var farm, server, id, title, secret, picUrl, items;
+
+            if (xmlhttp.readyState == 4 && xmlhttp.status == 200) {
+                var pics = [];
+                var myResp = JSON.parse(xmlhttp.response);
+                photoData(pics, myResp);
+                pupulate(pics);
+                items = document.getElementById('img0');
+                removeActiveAddCurrent(items);
+                getCarousel();
+            }
+        };
+        xmlhttp.open("Get", url, true);
+        xmlhttp.send();
+    }
+    function getCarousel(){
+        var box, next, counter, prev, items, amount, current;
+
+        next = document.getElementById('next');
+        counter = 0;
+        prev = document.getElementById('prev');
+        items = document.getElementsByTagName('img');
+        amount = items.length;
+        current = document.getElementsByClassName('current');
+
+        function navigate(direction){
+            if(direction === 1){
+                items = document.getElementById('img'+counter);
+                current = document.getElementsByClassName('current');
+                removeCurrentAddActive(items);
+            }
+        }
+        next.addEventListener("click", function(event){
```

Comment:
The `event` param is the proper attribute here, but since you're not using it within the handler, it isn't needed in the listener callback. Same for prev button.